package.json

```diff
@@ -1,5 +1,5 @@
 {
-  "name": "react-native-full-screen",
+  "name": "@windastella/react-native-full-screen",
```

Comment:
What is the reason to change the package name to a pull request? don't make sense.